src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -417,12 +421,13 @@ export const BorrowInput = ({
     return (
       <TradeAssetSelect
         assetId={collateralAssetId}
+        assetIds={collateralAssetIds}
```

Comment:
The actual fix for narrowing down collateral assets - we previously didn't need this because the narrowing is happening on the modal opened `onClick` - but now that we have related assets, these should be narrowed too